.travis.yml

```diff
@@ -13,6 +13,9 @@ script:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
   - julia -e 'Pkg.clone(pwd()); Pkg.checkout("DataFrames", "dfk/statsmodel-purge"); Pkg.build("StatsModels"); Pkg.test("StatsModels"; coverage=true)'
 after_success:
+  # build and deploy documentation with Documenter.jl
+  - openssl aes-256-cbc -K $encrypted_a9d536040c76_key -iv $encrypted_a9d536040c76_iv -in docs/.documenter.enc -out docs/.documenter -d
+  - julia -e 'cd(Pkg.dir("StatsModels")); Pkg.clone("https://github.com/MichaelHatherly/Documenter.jl"); include(joinpath("docs", "make.jl"))'
```

Comment:
Rather do a `Pkg.add("Documenter")` here instead of cloning from my personal fork of the package since it'll be a bit unstable and buggy more than likely.